CO2_Cleanup.ipynb

```diff
@@ -0,0 +1,149 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    from preparation import fft_mag, inv_fft, calc_freq
</pre><p>These functions are never used all produce errors that have to be dealt with. Meaning your data is not suitable for fft as discussed in class</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1-24-HW5/pull/61/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>